docs/content/docs/build/containers.md

```diff
@@ -3,28 +3,139 @@ title: Containers
+
+The container exposes functionality that allow cleanup of the container object. Calling dispose will remove any server connections and clean up registered events. Once a container is dispose it cannot be re-hydrated. Connecting to the same container requires another `getContainer(...)` call.
+
+```typescript
+container.dispose();
+
+const disposed = container.disposed();
+
+container.on("disposed", () => {
+    // handle event cleanup to prevent memory leaks
+});
+```
+
+### initialObjects
+
+`initialObjects` are the base set of `LoadableObjects` in a container. They exist for the lifetime of the container and are guaranteed to be available and loadable. They are defined via the container schema and match the signature on the schema. 
+
+For more on `initialObjects` see [Data modeling](data-modeling.md)
+
+### create
+
+The container also exposes a create function that allows dynamic `LoadableObject` creation.
+
+For more on dynamic object creation see [Data modeling](data-modeling.md)
 
 ## Patterns for managing container lifecycle
 
-### Create should be separate from load
+### Create/load separation
+
+When creating and loading a container our basic guidance is to separate the two flows. This provides a cleaner separation of responsibilities within the code it self. From a scenario perspective it manifests as the creator going through a explicit creation process that results in a redirect to a new page who's sole responsibility is to load the container. All users will load the container through this subsequent flow.
 
-## Advanced container patterns
+It should be noted that the downside of this approach is that the connection needs to be established twice. Once for the creation and once for the load. Depending on your scenario you should choose the pattern that works best for you.
 
 ### Multi-container example
+
+Multiple containers can be loaded from an app, or on a page, at the same time. There are two primary scenarios where you application would be using multiple containers.
+
+First, is if your application loads two different experiences that have different underlying data structures. `Experience 1` may require a `SharedMap` and `Experience 2` may requires a `SharedString`. To minimize our memory footprint we can create two different container schemas and load only what we need. In this scenario our app can load two different containers but we are only choosing to load one at a time.
+
+The second scenario involves loading two containers at once. Currently, all services enable permissioning at the container level, so we can use containers as a natural boundary for restricting access. An example scenario for this would be building an education app where mutliple teachers are gathering with students. The students and teachers may have a shared view while the teachers may also want to have an additional private view on the side. In this scenario the students would be loading one container and the teachers would be loading two.
```

Comment:
The only thing I don't like is stating that containers serve as a auth boundary. Containers can be used as a auth boundary but it's dependent on the service to support it and the developer to enable it. If the user doesn't auth correctly at the container level there's nothing we can do about it.

I used the word permissioning because you need specific a token for each container you are creating/loading. In this sense we are implying a potential auth boundary but not stating it. 

I would be much more comfortable talking about auth in the context of the frs docs because we can explicitly say if you implement this specific auth flow you can achieve this scenario. But as the generic client platform we can't make such garuantees.